### PR TITLE
UI/LOOP-935/Calibration-view

### PR DESCRIPTION
**Pull request by nhamming**
https://tidepool.atlassian.net/browse/LOOP-935

extensions to support Dexcom UI updates.

### CODE REVIEW

File: LoopKitUI/Views/TableViewTitleLabel.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//  Copyright © 2020 Tidepool Project. All rights reserved.
+//
+
+import UIKit
+
+public class TableViewTitleLabel: UILabel {
+    
+    override public init(frame: CGRect) {
+        super.init(frame: frame)
+        font = UIFont.titleFontGroupedInset
```
Comment (darinkrauss) [inline LoopKitUI/Views/TableViewTitleLabel.swift]: [nit] Perhaps a common `initFonts` function?

**Review by darinkrauss — APPROVED** [review]
Only one *really* minor nit/thought. LGTM. :shipit:

**Review by darinkrauss — APPROVED** [review]
LGTM 🚀

File: LoopKitUI/Extensions/UIFont.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+//  UIFont.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-01-17.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+extension UIFont {
+    public static var titleFontGroupedInset: UIFont {
+        return UIFont.systemFont(ofSize: 28, weight: UIFont.Weight.semibold)
```
Comment (ps2) [inline LoopKitUI/Extensions/UIFont.swift]: This does not respond to dynamic type. Perhaps take a look at https://developer.apple.com/documentation/uikit/uifont/1619030-preferredfont

**Review by ps2 — CHANGES_REQUESTED** [review]
We need to be able to respond to dynamic type in our interfaces for accessibility.

**Review by mpangburn — APPROVED** [review]
LGTM!

**Pull request merged**